src/app/fdctl/config.c

```diff
@@ -469,220 +470,365 @@ username_to_uid( char * username ) {
+  if( FD_UNLIKELY( affinity_tile_cnt<topo->tile_cnt ) )
+    FD_LOG_ERR(( "The topology you are using has %lu tiles, but the CPU affinity specified in the config tile as [layout.affinity] only provides for %lu cores. "
+                 "You should either increase the number of cores dedicated to Firedancer in the affinity string, or decrease the number of cores needed by reducing "
+                 "the total tile count. You can reduce the tile count by decreasing individual tile counts in the [layout] section of the configuration file.",
+                 topo->tile_cnt, affinity_tile_cnt ));
+  if( FD_UNLIKELY( affinity_tile_cnt>topo->tile_cnt ) )
+    FD_LOG_WARNING(( "The topology you are using has %lu tiles, but the CPU affinity specified in the config tile as [layout.affinity] provides for %lu cores. "
+                     "Not all cores in the affinity will be used by Firedancer. You may wish to increase the number of tiles in the system by increasing "
+                     "individual tile counts in the [layout] section of the configuration file.",
+                     topo->tile_cnt, affinity_tile_cnt ));
+
+  /*                                                                           topo, tile_name, tile_kind_id, fseq_wksp,   link_name,      link_kind_id, reliable,   polled */
+  for( ulong i=0; i<config->layout.net_tile_count; i++ )    fd_topob_tile_in(  topo, "net",     i,            "metric_in", "netmux_out",   0UL,          UNRELIABLE, POLLED ); /* No reliable consumers of networking fragments, may be dropped or overrun */
+  for( ulong i=0; i<config->layout.net_tile_count; i++ )    fd_topob_tile_in(  topo, "netmux",  0UL,          "metric_in", "net_netmux",   i,            UNRELIABLE, POLLED ); /* No reliable consumers of networking fragments, may be dropped or overrun */
+  for( ulong i=0; i<config->layout.verify_tile_count; i++ ) fd_topob_tile_in(  topo, "netmux",  0UL,          "metric_in", "quic_netmux",  i,            UNRELIABLE, POLLED ); /* No reliable consumers of networking fragments, may be dropped or overrun */
+  /**/                                                      fd_topob_tile_in(  topo, "netmux",  0UL,          "metric_in", "shred_netmux", 0UL,          UNRELIABLE, POLLED ); /* No reliable consumers of networking fragments, may be dropped or overrun */
+  for( ulong i=0; i<config->layout.verify_tile_count; i++ ) fd_topob_tile_in(  topo, "quic",    i,            "metric_in", "netmux_out",   0UL,          UNRELIABLE, POLLED ); /* No reliable consumers of networking fragments, may be dropped or overrun */
+  for( ulong i=0; i<config->layout.verify_tile_count; i++ ) fd_topob_tile_out( topo, "quic",    i,                         "quic_netmux",  i                                );
+  for( ulong i=0; i<config->layout.verify_tile_count; i++ ) fd_topob_tile_in(  topo, "verify",  i,            "metric_in", "quic_verify",  i,            UNRELIABLE, POLLED ); /* No reliable consumers, verify tiles may be overrun */
```

Comment:
I like `RELIABLE` and `POLLED` much better than just `1` like it was before, but it should probably be prefixed like `FD_TOPOB_POLLED` since it's in `disco` to avoid collisions. Since that'll make these lines even longer, consider declaring `net_cnt` and `verify_cnt` as local variables or something.